integration/unfork/istio-k8s/metadata.yaml

```diff
@@ -1,3 +1,3 @@
 args: ["--prefer-git"]
 upstream: https://github.com/istio/istio/tree/1.0.3/install/kubernetes/helm/istio
-fork: https://github.com/replicatedhq/test-charts/blob/d05895fe4fc808b82a46a856dfda11e3b19feffb/istio-forked/rendered.yaml
+fork: https://github.com/replicatedhq/test-charts/blob/dc51b6517eb333d6ac7cc096c9a058dd278b1cee/istio-forked/rendered.yaml
```

Comment:
didn't realize i hadn't merged this and i squashed and merged - hence the commit sha